docker-assets/manageiq.conf

```diff
@@ -1,9 +1,6 @@
 # Timeout: The number of seconds before receives and sends time out.
 Timeout 120
 
-# HTTP Start-up error log
-ErrorLog /var/log/httpd/miq_apache.log
```

Comment:
The file isn't even created, but there is an error_log in the directory already so no need to duplicate it.

There is no PV for this currently.